.gersemirc

```diff
@@ -0,0 +1,12 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/BlankSpruce/gersemi/master/gersemi/configuration.schema.json
+
+cache: true
+color: false
+definitions: []
+indent: 2
+line_length: 120
```

Comment:
why not 80 like we usually have?

Reply:
`.cmake-format` define it as 120 currently so I get the same length from there